modules/understanding-upgrade-channels.adoc

```diff
@@ -7,159 +7,145 @@
-be recommended upgrades.
-
-The candidate-{product-version} channel also allows upgrading from a previous minor version of
+After a version is available in the candidate channel, it goes through more quality checks, and if
+it meets the quality standard, it is promoted to the `fast-{product-version}` or `stable-{product-version}` channels.
+Because of this testing strategy, if a specific release is available in both the `candidate-{product-version}` channel and in the `fast-{product-version}`
+or `stable-{product-version}` channels, it is a Red Hat supported version.
+The `candidate-{product-version}` channel can include release versions that you can never upgrade from.
+
+The `candidate-{product-version}` channel also allows upgrading from a previous minor version of
 {product-title} to {product-version}.
 
 [NOTE]
 ====
-Release candidates differ from the nightly builds found on try.openshift.com. Nightly
-builds are available for early access to features but updating to or from nightly
-builds is neither recommended nor supported. Nightly builds are not avilable in
+Release candidates differ from the nightly builds found on the link:https://www.openshift.com/try site. Nightly
+builds are available for early access to features, but updating to or from nightly
+builds is neither recommended nor supported. Nightly builds are not available in
 any upgrade channel.
 ====
 
 [discrete]
-== fast-{product-version} Channel
+== fast-{product-version} channel
 
-The fast-{product-version} channel is updated with new {product-version} patch
-versions as soon as Red Hat declares the given patch as a general availability
-release. As such, these releases are fully supported, are production quality and have
-performed well while available as a release candidate in the candidate-{product-version}
-channel from where they were promoted. Some time after a release appears in
-fast-{product-version} it will also appear in stable-{product-version}. Releases will
-never appear in stable-{product-version} before they appear in fast-{product-version}.
+The `fast-{product-version}` channel is updated with new {product-version}
+versions as soon as Red Hat declares the given version as a general availability
+release. As such, these releases are fully supported, are production quality, and have
+performed well while available as a release candidate in the `candidate-{product-version}`
+channel from where they were promoted. Some time after a release appears in the
+`fast-{product-version}` channel, it is added to the `stable-{product-version}` channel. Releases
+never appear in the `stable-{product-version}` channel before they appear in the `fast-{product-version}` channel.
 
-The fast-{product-version} channel also allows upgrading from a previous minor version of
-{product-title}.
+By subscribing to the `fast-{product-version}` channel, you have the ability to upgrade from a previous minor version of {product-title}.
```

Comment:
Can we echo the earlier candidate wording for this?  That has "allows", which seems weaker to me than "have the ability to".  Although thinking about this more, I can't put my finger on why.  In any case, the idea we're trying to express for all of these channels is that they _may_ include both `{product-version}` releases and releases from the previous minor version of `{product-title}`.  We should avoid phrasing that sounds like we are committing to those channels _actually_ including releases from either channel or having any cross-minor update recommendations.  Both of those are possible, but not required.

Reply:
I'm happy to pull "allows" from all of them. @wking, PTAL?